dccrg.hpp

```diff
@@ -9701,7 +9708,7 @@ template <
 					this->neighborhood_of,
 					2 * this->max_ref_lvl_diff,
 					true
-				);
+				);	// todo PRETTY SURE THIS SHOULDN'T BE HERE!
```

Comment:
So should we discuss this comment?

Reply:
Right. I'm assuming that should be just getting neighbors instead of finding them but I wasn't sure due to that being in the middle of refinement.